GoogleForCreators/web-stories-wp
# Floating menu: Add Text formatting toggles

## Context



## Summary
Makes bold, italic, underline functional


## Relevant Technical Choices
Changed the `tabIndex` of the three toggles to `0` -- it was impossible to get into those with keyboard otherwise.
To be followed up in #10872 


## To-do



## User-facing changes
<img width="536" alt="Screenshot 2022-03-10 at 13 33 44" src="https://user-images.githubusercontent.com/3294597/157653756-10f0b0df-c2f6-4438-abae-20e53c7f2c05.png">



## Testing Instructions






This PR can be tested by following these steps:

1. Add a Text element
2. Verify the bold, italic, underline toggles work as expected.
3. Verify the toggles work when selecting only part of the text as well.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?

Adds tracking in the following format for each toggle:
```
    trackEvent('floating_menu', {
        name: 'set_bold',
        element: 'text',
      });
```



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #10631
